### DISCUSSION TITLE
Use for Non-Date-Polls

### DISCUSSION THREAD

**Discussion by githubkoma** [lukevella/rallly#148] [Ideas]
Nice Application you've created here.

Would be great if one could create polls about non-date topics.
Like in Framadate, where you can make an "event poll" or "classic poll".
Like to vote for where to order pizza or which movie to watch in cinema ;)

best regards,
chris

**Comment by heyvoon** [comment]
It would be very nic to be able to select a Time range by selected day.
Example.:
on a whole month I have people available 3 times a week but during different time each day. Would be nice for these people to be able to select on a daily basis witch time range they are available on their available dates.

Thank you and keep up the awesome work! ;-)

**Comment by sean-wils** [comment]
Ha, I literally just tried to create a non-date poll with Rallly.
I thought I'd be able to choose the type of poll it was on Step 2.

**Comment by tbleiker** [comment]
Will this be considered? I really like the app, but "classic polls" are something I do quite a lot. See for example bitpoll (https://bitpoll.mafiasi.de/).

**Reply by lukevella** [reply]
Yes, it will be added. The data model has been built with this feature in mind. I will work on adding support for it in the coming weeks.

**Reply by willjukes** [reply]
Discovered rallly last week when looking around for a Doodle Poll alternative for the office event organisers. I found Rallly and thought it was great but just missing the simple event list type polls discussed on this thread. Any idea when this feature might go live? If you haven't started work on it yet I might try and add myself?

**Reply by lukevella** [reply]
I'm working towards it but can't give an accurate estimate for when it will be available yet. Appreciate you offering to pick this up but it wouldn't be viable for this piece of work because there would be no way to do this without lengthy design reviews and major merge conflicts.

In the meantime… If you're looking for an alternative I recently discovered [tally.so](https://tally.so) which seems like it would be quite handy for this use case. Might be worth checking out.

**Reply by llann** [reply]
I hope that this feature might come at some point like in framadate. That's the main feature missing for me! But really nice project !! Thanks!

If you do it, can you please make it a bit more advanced than framadate and set an option to limit to only one choice between questions? Like in Doodle?

**Comment by 1000101** [comment]
Looking forward to this feature! It's the only thing that I'm currently missing.

**Comment by benjamin051000** [comment]
What's the progress on this feature? Is it still in the works? Is there a trello/kanban board where we can watch progress in general?

**Reply by lukevella** [reply]
Sorry I haven't made an update about this. When I built the current version of Rallly I had fully intended to support this feature but having thought about this a bit more I believe it's better to focus on the scheduling side of things rather than the polling. It may be possible to embed other questions inside a date poll but creating a standalone non-date poll is not something I consider within the scope of the product.

It's possible that a spin-off product could be created that focuses on generic polls but I don't have the bandwidth for that right now so I can't offer a time frame when this feature would be available. Also, the need for such a product is not clear since there are many great options available for creating polls.

**Reply by benjamin051000** [reply]
Gotcha. What I'm personally looking for is simply embedding normal-poll style questions inside a when-to-meet schedule poll. I don't need a stand-alone poll system, that's pretty different than what this project offers I think.

**Reply by benjamin051000** [reply]
Regardless, I _love_ the scheduling features, thank you for making this awesome app! I look forward to seeing where it goes 😄

**Reply by lukevella** [reply]
Thanks @benjamin051000, I think adding questions is definitely something you can expect. It will probably be a while longer before that's available though so maybe don't hold your breath.

If you would like to share what kind of questions you would like to be able to ask inside a scheduling poll I'd appreciate the insight.

**Reply by benjamin051000** [reply]
Mostly simple stuff like the original discussion comment: pick a time to go to the movies, also add a poll to vote for which movie you want to see.

Choose date for a party, and add a little question/response for what snacks you plan to bring. 

This way, all the necessary info is within one app. Currently I need this app to plan the date/time, and something like google forms to handle the other stuff. 

It doesn't need to be super feature rich or anything, but things I'd love to see are:

- Question/answer free response form
- single response poll
- checkbox poll (select all that apply, multiple response)

That would probably cover most users' bases for stuff like this

**Reply by lukevella** [reply]
Nice, I like the way you broke it down into 3 options (free form, single choice, multi choice). To answer your question from before, I don't have a public board to track progress. The development process is still a bit too chaotic for that but it's getting better and I do plan on sharing more of it as the project matures. I've noted your feedback though and will definitely take it into account when I start working on this.

**Comment by Pluckerpluck** [comment]
Just wanted to throw in my desire for this, recognising it's likely not a priority at all.

Trying to decide on a best "meet up organiser" to self-host and this was up there on the list thanks to mobile support. Just unfortunate it can't do generic selection polls to handle things like what film we want to go and see etc.

I do wonder how hard baked this is against it nowadays though. I do generally like this project and if I get time I might mess around with the codebase.... Not that I don't have a huge pile or projects I wish to work on elsewhere....
